godel_robot_config/urdf/sia20d_workspace.urdf

```diff
@@ -326,7 +326,7 @@
   <joint name="asus_mount_to_asus_link" type="fixed">
     <parent link="asus_mount"/>
     <child link="asus_link"/>
-    <origin rpy="3.14 0.0 0.0" xyz="0 0 0"/>
+    <origin rpy="-3.123031 0.001618 0.02739" xyz="0.01479 0.006741 0.007162"/>
```

Comment:
Those numbers came from the associated xacro file. I did not change them, merely ran xacro and updated this file. All of these are changing soon when I calibrate the Kinect2.